content/agent/autodiscovery.md

```diff
@@ -284,7 +287,7 @@ spec:
 The Agent detects if it's running on Docker and automatically searches all labels for check templates.
 
 Since version 6.2 of the Datadog Agent, it is also possible to configure Docker log collection in container labels.
-Check the [Docker Log collection guide][23] for more information about the setup.
+Check our [Docker Log collection guide][23] for more information about the setup.
```

Comment:
We don't [use the first-person](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md).